locales/zh_CN.json

```diff
@@ -79,7 +79,7 @@
     },
     "aria-roles": {
       "description": "确保所有元素包含的 role 属性值都是有效的",
-      "help": "使用的 ARIA 角色必须符合有效的值"
+      "help": "使用的 ARIA 角色属性值必须有效"
```

Comment:
Here is what I meant by inconsistence. Some of the `role`s are translated and some of them are not. So I hope the mapping at the very beginning would help a bit.

Reply:
I understand what you mean. Thank you for your review.